### PR TITLE
Add an effect for Emblazon Armament(Shield)

### PR DESCRIPTION
**Pull request by arthurtrumpet**

### CODE REVIEW

File: packs/feat-effects/effect-emblazon-armament-shield.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "ActiveEffectLike",
```
Comment (simonward) [inline packs/feat-effects/effect-emblazon-armament-shield.json:20]: Use ItemAlteration instead. Take a look at sapling shield for an example.

**Review by simonward — CHANGES_REQUESTED** [review]

File: packs/feats/emblazon-armament.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
...
+            "value": "<p>Carefully etching a sacred image into a physical object, you steel yourself for battle. You can spend 10 minutes emblazoning a symbol of your deity upon a weapon or shield. The symbol doesn't fade until 1 year has passed, but if you Emblazon an Armament, any symbol you previously emblazoned and any symbol already emblazoned on that item instantly disappears. The item becomes a religious symbol of your deity and can be used as a divine focus while emblazoned, and it gains another benefit determined by the type of item. This benefit applies only to followers of the deity the symbol represents.</p>\n<ul>\n<li><strong>@UUID[Compendium.pf2e.feat-effects.Item.o04gwnrVtzWyIEs8]{Shield}</strong> The shield gains a +1 status bonus to its Hardness. (This causes it to reduce more damage with the Shield Block reaction.)</li>\n<li><strong>@UUID[Compendium.pf2e.feat-effects.Item.Effect: Emblazon Armament (Weapon)]{Weapon}</strong> The wielder gains a +1 status bonus to damage rolls.</li>\n</ul>"
```
Comment (simonward) [inline packs/feats/emblazon-armament.json:14]: This should link to the effect item name rather than ID. It will be converted to build time.

Comment (arthurtrumpet) [reply packs/feats/emblazon-armament.json]: Extracted packs in correct order, and it now uses the name not ID

File: packs/feat-effects/effect-emblazon-armament-shield.json (hunk 1 above)
Comment (arthurtrumpet) [reply packs/feat-effects/effect-emblazon-armament-shield.json]: Updated effect to use choice set to select shield, then ItemAlteration to upgrade hardness

File: packs/feat-effects/effect-emblazon-armament-shield.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            {
+                "itemType": "armor",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "item:id:{item|flags.pf2e.rulesSelections.effectEmblazonArmamentShield}"
+                ],
+                "property": "hardness",
+                "type": "status",
```
Comment (stwlam) [inline packs/feat-effects/effect-emblazon-armament-shield.json:38]: ?

Comment (arthurtrumpet) [reply packs/feat-effects/effect-emblazon-armament-shield.json]: Removed type status from effect, as not valid field in ItemAlteration

**Comment by arthurtrumpet** [comment]
Implemented all review comments

File: packs/feat-effects/effect-emblazon-armament-shield.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+                "mode": "add",
+                "predicate": [
+                    "item:id:{item|flags.pf2e.rulesSelections.effectEmblazonArmamentShield}"
+                ],
+                "property": "hardness",
+                "value": 1
+            }
+        ],
+        "source": {
+            "value": ""
```
Comment (TikaelSol) [inline packs/feat-effects/effect-emblazon-armament-shield.json:42]:
```suggestion
            "value": "Pathfinder Core Rulebook"
```

**Pull request merged**